notebooks/06_Serving/06_02_Kfserving.ipynb

```diff
@@ -0,0 +1,493 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define InferenceService\n",
+    "Firstly define default endpoint spec, and then define the inferenceservice basic on the endpoint spec."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION\n",
+    "default_endpoint_spec = V1alpha2EndpointSpec(\n",
+    "                          predictor=V1alpha2PredictorSpec(\n",
+    "                            tensorflow=V1alpha2TensorflowSpec(\n",
+    "                              storage_uri='gs://kfserving-samples/models/tensorflow/flowers',\n",
+    "                              resources=V1ResourceRequirements(\n",
+    "                                  requests={'cpu':'100m','memory':'1Gi'},\n",
+    "                                  limits={'cpu':'100m', 'memory':'1Gi'}\n",
+    "                              )\n",
+    "                            )\n",
+    "                          )\n",
+    "                        )\n",
+    "    \n",
+    "isvc = V1alpha2InferenceService(\n",
+    "        api_version=api_version,\n",
+    "        kind=constants.KFSERVING_KIND,\n",
+    "        metadata=client.V1ObjectMeta(name='flower-sample', namespace=namespace),\n",
+    "        spec=V1alpha2InferenceServiceSpec(default=default_endpoint_spec)\n",
+    "       )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create InferenceService\n",
+    "Call KFServingClient to create InferenceService."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KFServing = KFServingClient()\n",
+    "KFServing.create(isvc)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Check the InferenceService"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "KFServing.get('flower-sample', namespace=namespace, watch=True, timeout_seconds=120)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Invoke Endpoint\n",
+    "\n",
+    "If you want to invoke endpoint by yourself, you can copy and paste below code block and execute in your local environment. Remember you should have a `input.json` file in the same directory when you execute.  \n",
+    "\n",
+    "Note: Replace `<YOUR-CURRENT-NAMESPACE>` with your current namespace. The current namespace is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "namespace"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MODEL_NAME=flower-sample\n",
```

Comment:
Seems this is shell scripts? right? Is it runnable here?  Should you have a %bash header?

Reply:
Yes, but the point is I tell users to execute these code in their local environment, since in their namespace, they have no access get resources from `istio-system` namespace. 

One example:

![image](https://user-images.githubusercontent.com/23116624/81220454-2a8d6f80-8f96-11ea-8986-021139b8b806.png)

Reply:
So I added %%bash, but users still need to execute code snippet in their local env.